### ISSUE TITLE
[test-triage] chip_sw_uart_tx_rx_alt_clk_freq timing out

### ISSUE THREAD

**Issue by engdoreis** [lowRISC/opentitan#18142]
### Hierarchy of regression failure

Chip Level

### Failure Description

```
Job chip_earlgrey_asic-sim-vcs_run_default killed due to: Exit reason: User job exceeded runlimit: User job timed out
```

### Steps to Reproduce

- GitHub Revision: [2c75f3a50](https://github.com/lowrisc/opentitan/tree/2c75f3a500fd88d3e143bd25a988f3ae37cfdcc1)
- dvsim invocation command to reproduce the failure, inclusive of build and run seeds:
```sh
./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i chip_sw_uart_tx_rx_alt_clk_freq \
--build-seed 3316792930 --fixed-seed 3159293720 --waves fsdb
```

### Tests with similar or related failures

- [ ] chip_sw_uart_tx_rx_alt_clk_freq

**Comment by johngt** [comment]
This is a timeout test that could be re-run on a H cloud instance and checked to see if the test passes there. If it does - can then safely close this out.

**Comment by a-will** [comment]
Fixed by #18203

Sorry I didn't notice this issue was filed.
